src/inc/quic_driver_helpers.h

```diff
@@ -82,7 +82,7 @@ class QuicDriverService {
                     DriverFilePath,
                     nullptr,
                     nullptr,
-                    "msquic\0",
+                    "msquicpriv\0",
```

Comment:
This is going to break Windows tests as they depend on the real msquic.

Reply:
Hmm. Thats an interesting issue. We'll have to have a separate user mode executable too. I can do that easily enough.